src/styles/blocks/card.scss

```diff
@@ -0,0 +1,100 @@
+@import "../utils/vars";
+
+.card {
+  max-width: $card-width;
+  padding: 32px 16px 16px;
+  background-color: $white-color;
+  border: $card-border;
+  border-radius: 5px;
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    display: block;
+    margin: 0 3px 40px;
+  }
+
+  &__title {
+    color: $primary-color;
+    font-family: inherit;
+    font-size: $font-size-sm;
+    line-height: $line-height-lg;
+    font-weight: $font-weight-medium;
+    margin-bottom: 4px;
+  }
+
+  &__prodcode {
+    color: $secondary-color;
+    font-family: inherit;
+    font-size: $font-size-xs;
+    line-height: $line-height-sm;
+    font-weight: $font-weight-light;
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__rating-reviews {
+    color: $primary-color;
+    font-family: inherit;
+    font-weight: $font-weight-light;
+    font-size: $font-size-xs;
+    line-height: $line-height-sm;
+  }
+
+  &__price-label {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__price {
+    margin-bottom: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price-text {
+    color: $secondary-color;
+    font-family: inherit;
+    font-size: $font-size-sm;
+    line-height: $line-height-lg;
+    font-weight: $font-weight-light;
+  }
+
+  &__price-number {
+    color: $primary-color;
+    font-family: inherit;
+    font-size: $font-size-lg;
+    line-height: $line-height-lg;
+    font-weight: $font-weight-bold;
+  }
+
+  &__btn {
+    display: block;
+    text-transform: uppercase;
+    color: $white-color;
+    font-family: inherit;
+    text-decoration: none;
+    text-align: center;
+    font-size: $font-size-md;
+    font-weight: $font-weight-bold;
+    background-color: $theme-color;
+    line-height: $line-height-md;
+    padding: 11px 0;
+    border-radius: 5px;
+    transition: 0.3s all ease;
```

Comment:
Don`t use "all" for transition, use property that will change. Write all values in the correct order:
transition: property, duration, transition-timing-function;

Place transition duration in variable